src/en/mangago/src/eu/kanade/tachiyomi/extension/en/mangago/Mangago.kt

```diff
@@ -473,4 +497,20 @@ class Mangago : ParsedHttpSource() {
             )
         }
     }
+    private fun isRemoveTitleVersion() = preferences.getBoolean(REMOVE_TITLE_VERSION_PREF, false)
+
+    override fun setupPreferenceScreen(screen: PreferenceScreen) {
+        SwitchPreferenceCompat(screen.context).apply {
+            key = REMOVE_TITLE_VERSION_PREF
+            title = "Remove version information from entry titles"
+            summary = "This removes version tags like “(Official)” or “(Yaoi)” from entry titles " +
```

Comment:
Avoid smart quotes

Reply:
sorry (this is probably a dumb question) but what is smart quotes exactly?

Reply:
By smart quotes I mean the angled quotes, not sure if it goes by another name. It's dependent on your locale so I'd say it's better to avoid.

Reply:
oh okay ty

Reply:
![image](https://github.com/user-attachments/assets/01575eb4-0810-4703-818e-20871a9f8834)
is it good like this?

Reply:
Either `'` or `"` is good, so that looks fine.